arch/arm/src/armv8-m/svcall.h

```diff
@@ -122,6 +115,17 @@
 #define SYS_signal_handler_return (7)
 
 #endif /* CONFIG_BUILD_PROTECTED */
+
+/* SYS call 5:
+ *
+ * void up_pthread_start(pthread_startroutine_t startup,
+ *                       pthread_startroutine_t entrypt, pthread_addr_t arg)
+ *        noreturn_function
+ */
+
+#define SYS_pthread_start         (5)
```

Comment:
where SYS_pthread_exit

Reply:
CONFIG_SYS_RESERVED need update

Reply:
> where SYS_pthread_exit

It is a normal system call.  SYS_pthread_start is not.  The start function is an internal chip-specific OS interface, SYS_pthread_exit (or is it SYS_nxpthread_exit?).  Is part of the normal user interface.

Reply:
I verified that the code is correct as it is. The PROTECTED build stm32f4discovery:kostest works correctly as is, but fails if this recommended change is added.